scripts/medium.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+docker run -d --name elasticsearch -p 9200:9200 -p 9300:9300 elasticsearch
+SNAP_ELASTICSEARCH_HOST=$(docker inspect -f '{{ .NetworkSettings.IPAddress }}' elasticsearch)
```

Comment:
This will not work if someone is using docker engine, without docker machine. I suggest you use $DOCKER_HOST instead, see https://github.com/intelsdi-x/snap-plugin-collector-bind/blob/master/scripts/medium.sh#L22-L28